### PR TITLE
Add query validation

### PR DESCRIPTION
**Pull request by BradLewis**
Added query validation to easily validate all the query parameters. Set some sane defaults to start with. 
I also moved the font and fallback image to the assets folder and fixed some function names from `snake_case` to `camelCase`

### CODE REVIEW

File: cmd/api.go [hunk 1]
```diff
@@ -9,18 +9,40 @@ import (
...
+	Username string `url:"username" validate:"required"`
+	Period   Period `url:"period" validate:"required,validatePeriod"`
 }
 
-func get_collage(request *CollageRequest) image.Image {
-	limit := request.Rows * request.Columns
-	albums := get_albums(request.Username, request.Period, limit)
+func getCollage(request *CollageRequest) image.Image {
+	count := request.Rows * request.Columns
+	albums := get_albums(request.Username, request.Period, count)
```
Comment (TheDen) [inline cmd/api.go:45]: I think should be `getAlbums`

Comment (TheDen) [reply cmd/api.go:45]: Same on line `52` with `create_collage`

Comment (BradLewis) [reply cmd/api.go]: Good catch! I ran the rename but the `api.go` file didn't get saved lol

**Review by TheDen — APPROVED** [review]
LGTM 👍

**Pull request merged**